zinc/src/sbt-test/source-dependencies/java-class-invs/test

```diff
@@ -0,0 +1,4 @@
+> use/compile
+$ sleep 1000
```

Comment:
Why is the sleep necessary?

Reply:
It's a last-modified-time-granularity thing.  There's a comment associated to the first instance of it in the test suite, which I've since shed.  I've been thinking it should be handled intrinsically in scripted's compilation where, additionally, it can be both manually and globally tweaked locally as well as intelligently configured (like sbt.io.IO does, IIRC).